src/components/StatsInDetail/Charts/OverallTrendsChart.tsx

```diff
@@ -54,30 +55,34 @@ const Chart: React.FC<ChartProps> = ({ csvPath, title, width = '100%', height =
                 },
             },
             xAxis: {
-                type: 'category',
-                data: chartData.dates.map((date) => date.monthYear),
+                type: 'time',
+                data: chartData.dates,
                 axisLabel: {
-                    formatter: (_value: unknown, index: number) => {
-                        const year = chartData.dates[index].year
-                        return (index === 0 || year !== chartData.dates[index - 1].year) && parseInt(year) % 2 === 0
-                            ? year
-                            : ''
+                    formatter: (value: string) => {
+                        const date = new Date(value)
+                        const year = date.getFullYear()
+                        return year.toString()
                     },
-                    interval: 0,
                 },
-                axisTick: { show: false },
+                axisTick: { show: true, alignWithLabel: true },
             },
             yAxis: {
                 type: 'value',
                 splitLine: { lineStyle: { type: 'dashed' } },
             },
             series: [
                 {
-                    data: chartData.values,
+                    data: chartData.dates.map((date, index) => [date, chartData.values[index]]),
```

Comment:
Is this related to responsiveness? If not I'd suggest removing this change from this PR and create another one.

Same for the areaStyle below.

Reply:
Should heed this suggestion moving forwards, but I suspect the plot would not render properly without this, so will make an exception for it this time.